llvm/lib/Transforms/Utils/SimplifyCFG.cpp

```diff
@@ -5179,6 +5184,15 @@ bool SimplifyCFGOpt::simplifyUnreachable(UnreachableInst *UI) {
 
   bool Changed = false;
 
+  // Ensure that any debug-info records that used to occur after the Unreachable
```

Comment:
A few lines up there's an ominous comment `// WARNING: keep in sync with InstCombinerImpl::visitUnreachableInst()!` - does anything over there need changing?